pkg/cmd/pulumi/package_publish.go

```diff
@@ -162,17 +162,15 @@ func publishToNPM(path string) error {
 		// to verify we're not encountering a time-of-check to time-of-use (TOC/TOU) issue.
 		infoCheckCmd := exec.Command("npm", "info", pkgNameWithVersion)
 		infoCheckCmd.Stderr = os.Stderr
-		checkOutput, checkErr := infoCheckCmd.Output()
-		if checkErr != nil {
-			return fmt.Errorf("running npm info to verify failed %w", checkErr)
-		}
+		checkOutput, _ := infoCheckCmd.Output()
+
```

Comment:
Mind adding another copy of that comment here?
Just something to clarify that the error ignore is desirable.

```
// Ignore error. stdout will be empty if the package was not published.
```